src/core/ProjectRenderer.cpp

```diff
@@ -37,55 +37,54 @@
 #include "sched.h"
 #endif
 
+
 const ProjectRenderer::FileEncodeDevice ProjectRenderer::fileEncodeDevices[] =
 {
-
-	{ ProjectRenderer::WaveFile,
-		QT_TRANSLATE_NOOP( "ProjectRenderer", "WAV-File (*.wav)" ),
-					".wav", &AudioFileWave::getInst },
-	{ ProjectRenderer::FlacFile,
-		QT_TRANSLATE_NOOP("ProjectRenderer", "FLAC-File (*.flac)"),
-		".flac",
-		&AudioFileFlac::getInst
+	{ProjectRenderer::WaveFile,
```

Comment:
We need to stop translating stuff like `*.mp3`.  This is only going to cause issues down the road where a bad translation breaks a feature.

Reply:
Yeah according to #3677 this is already making problems in other places

Reply:
I just removed the space after `{` here.
Do you want me to keep the strings but make them untranslatable? If so, is removing `QT_TRANSLATE_NOOP` enough?